src/maicos/lib/math.py

```diff
@@ -507,7 +507,10 @@ def center_cluster(ag: mda.AtomGroup, weights: np.ndarray) -> np.ndarray:
 
 
 def symmetrize(
-    m: np.ndarray, axis: None | int | tuple[int] = None, inplace: bool = False
+    m: np.ndarray,
+    axis: None | int | tuple[int] = None,
+    inplace: bool = False,
+    parity: str = "even",
```

Comment:
This is really a bool. I would call it `is_even`. Currently, I can pass any str and the profile will be treated as `"odd"`

Reply:
Makes sense 👍